contrib/Intel/JenkinsBuildScripts/tests.py

```diff
@@ -0,0 +1,311 @@
+import sys
+import os
+
+print(os.environ['CI_SITE_CONFIG'])
+sys.path.append(os.environ['CI_SITE_CONFIG']) # for adding path for cisite_config
+
+import subprocess
+import re
+import cisite_config
```

Comment:
I think I ll do ci_site_config since it was conflicting with the existing site_config which is in the python path and removing that would create problems for build bot